package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "HTML5 skin over an Ooyala video player",
   "engines": {
-   "node" : ">=0.10.33 <=0.10.46"
+    "node": ">=8.11.3"
```

Comment:
This is not needed yet, but we probably should look into using  `package-lock.json` instead of shrinkwrap now that we have a node version that supports it. Not sure if it's worth the effort though, just thinking out loud.